docs/getting-started/README.md

```diff
@@ -0,0 +1,95 @@
+# Getting started with Triggers
+
+To get started with Triggers, lets put it to work building and deploying a real image. In the following guide we will use `Triggers` to handle a real Github webhook request to kickoff a PipelineRun.
+
+## Install dependencies
+
+Before we can use the Triggers project, we need to get some dependencies out of the way.
+
+  - [A Kubernetes Cluster](https://kubernetes.io/docs/setup/)
+    - For our purposes any publicly available k8s cluster will do, as long as it can produce a Service Loadbalancer.
+  - [Install Tekton Pipelines](https://github.com/tektoncd/pipeline/blob/master/docs/install.md#installing-tekton-pipelines)
+    - Pipelines is the backbone of Tekton and will allow us to accomplish the work we plan to do.
+  - [Install Triggers](https://github.com/tektoncd/triggers/blob/master/DEVELOPMENT.md#install-triggers)
+    - Of course we need to install our project as well, so we can accept and process events into PipelineRuns!
+  - Pick a Github repo with a Dockerfile as your build object (or you can fork [this one](https://github.com/iancoffey/ulmaceae)).
+    - Clone this repo locally -  we will come back to this repo later.
+## Configure the cluster
+
+Now that we have our cluster ready, we need to setup our getting-started namespace and RBAC. We will keep everything inside this single namespace for easy cleanup. In the unlikely event that you get stuck/flummoxed, the best course of action might be to just delete this namespace and start fresh.
+
+- Create the *getting-started* namespace, where all our resources will live:
+  - `kubectl create namespace getting-started`
+- [Create the admin user, role and rolebinding](./rbac/admin-role.yaml)
+  - `kubectl apply -f ./docs/getting-started/rbac/admin-role.yaml`
+- [Create the create-webhook user, role and rolebinding](./rbac/webhook-role.yaml)
+  - `kubectl apply -f ./docs/getting-started/rbac/webhook-role.yaml`
+  - This will allow our webhook to create the things it needs to.
+
+## Install the Pipeline and Triggers
+
+Now we have to install the Pipeline we plan to use and also our Triggers resources.
+
+Our Pipeline will do a few things:
+- Retrieve the source code
+- Build and push the source code into a docker image
+- Push the image to the specified repository
+- Run the image locally
+<!-- -->
+- [Install the Pipeline](./pipeline.yaml)
+  - The Pipeline will build a docker image with img and deploy it locally via kubectl image.
+  - `kubectl apply -f ./docs/getting-started/pipeline.yaml`
+<!-- -->
+Our Triggers project will pickup from there:
+- We will setup an EventListener to accept and process Github Push events
+- A TriggerTemplate to create templates Resources and PipelineRun per event received.
```

Comment:
Nit: Backticks on the Triggers resources?
I feel like it's probably better for the sake of consistency with the rest of the docs

Reply:
makes sense, will update